src/v/security/audit/schemas/utils.cc

```diff
@@ -590,4 +590,16 @@ make_application_lifecycle(application_lifecycle::activity_id activity_id) {
       create_timestamp_t<std::chrono::system_clock>()};
 }
 
+application_lifecycle make_application_lifecycle(
+  application_lifecycle::activity_id activity_id, ss::sstring feature_name) {
+    auto product = redpanda_product();
+    product.feature = feature{.name = std::move(feature_name)};
+
+    return {
+      activity_id,
+      std::move(product),
+      severity_id::informational,
+      create_timestamp_t<std::chrono::system_clock>()};
```

Comment:
I didn't review the PR that brought this in, but I'm finding it hard to convince myself that `time_stamp_t` and the associated machinery shouldn't just be based on `ss::lowres_system_clock` that has a rep of milliseconds, and then just use the std::chrono library:

```c++
namespace audit {
class system_clock final {
    using impl = ss::lowres_system_clock;
public:
    using duration = std::chrono::milliseconds;
    using rep = duration::rep;
    using period = duration::period;
    using time_point = std::chrono::time_point<system_clock, duration>;
    static constexpr bool is_steady = impl::is_steady;
    static time_point now() noexcept {
        return std::chrono::duration_cast<duration>(impl::now());
    }

    static std::time_t to_time_t(time_point t) noexcept {
        return impl::to_time_t(t);
    }

    static time_point from_time_t(std::time_t t) noexcept {
        return impl::from_time_t(t);
    }
};
}
```
Caveat emptor; I didn't test this.

Reply:
>std::chrono::system_clock

good point. we should avoid the std library system clock unless we have a good reason to use it. consider using `lowres_system_clock`.

Reply:
Sure I can replace all of the `std::chrono::system_clock` with `ss::lowres_system_clock`.  I don't know if I need to go the full route of creating a new class for it.  `timestamp_t` is just a signed 64-bit value.

Reply:
> Sure I can replace all of the `std::chrono::system_clock` with `ss::lowres_system_clock`.  I don't know if I need to go the full route of creating a new class for it.  `timestamp_t` is just a signed 64-bit value.

Right, but without all the support of the type system of  `std::chrono`. I'm not suggesting make the change in this PR, but I don't think it's particularly onerous to plug into std::chrono and keep it type-safe and familiar.